src/System.Windows.Forms/src/System/Windows/Forms/ToolTip.cs

```diff
@@ -2046,52 +1944,55 @@ private void Reposition(Point tipPosition, Size tipSize)
             NativeMethods.SWP_NOACTIVATE | NativeMethods.SWP_NOSIZE | NativeMethods.SWP_NOOWNERZORDER);
         }
 
+        private IntPtr GetCurrentToolHwnd()
+        {
+            var toolInfo = new ComCtl32.ToolInfoWrapper();
+            if (toolInfo.SendMessage(this, WindowMessages.TTM_GETCURRENTTOOLW) != IntPtr.Zero)
```

Comment:
For consistency sake:
```suggestion
            var result = toolInfo.SendMessage(this, WindowMessages.TTM_GETCURRENTTOOLW);
            if (result != IntPtr.Zero)
```

Reply:
I don't think that's useful- it's pretty standard to check results in-line if you don't need to do additional logic with it. If I did, it should not be var.

Reply:
I put `var` because the browser doesn't have intellisense :)